### ISSUE TITLE
Opera & Chrome support

### ISSUE THREAD

**Issue by Riajyuu** [eric-bixby/auto-sort-bookmarks-webext#38]
Is it possible to port this extension to Opera and Google Chrome?
Though API restriction can be an issue.

**Comment by eric-bixby** [comment]
The auto-sort-bookmarks add-on uses the Mozilla API, which is not currently supported by Opera nor Google Chrome.  However, I haven't tried it, but apparently, you can port Google Chrome and Opera extensions (WebExtensions) to Firefox:
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Porting_a_Google_Chrome_extension

For Google Chrome, try Sprucemarks:
https://github.com/ForestMist/sprucemarks

**Comment by smaragdus** [comment]
@eric-bixby 

Sprucemarks has become commercial extension and its GitHub page was deleted. I am not aware of a really good alternative to Auto-Sort Bookmarks for Chromium browsers- all I have checked are either abandoned long time ago or are very primitive. I would like to see Auto-Sort Bookmarks ported to Chromium in the future if possible as I have switched from Firefox to a Chromium fork long ago.

**Comment by eric-bixby** [comment]
Yes, I’ve been thinking about doing that. I was bummed that Sprucemark is no longer free and experimented with a Chrome version before but didn’t continue because I was content with Sprucemrk. Also, Edge is now chrome-based.

My only concern is that I recently read Google wants to charge a fee for publishing to the Google store.

**Comment by smaragdus** [comment]
@eric-bixby 

Thanks for the quick response.

I am just a user but another developer told me that Google really charges developers for registration (it is hardly believable how greedy Google is). The same developer told me that Microsoft does not charge developers on their store.

If you port your extension to Chromium you can release it as CRX or ZIP file here on GitHub. Additionally you can also publish it on Edge store- you won't be charged for that.

Regards
